CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+## 0.1.0 (Unreleased)
+
+ENHANCEMENTS:
+
```

Comment:
Maybe we could write something here already, since if we are adding a CHANGELOG.md and the terms, should they be the same as in https://changelog.md example here https://keepachangelog.com/en/1.0.0/

Like:

```
# Changelog
All notable changes to this project will be documented in this file.
See updating [Changelog example here](https://keepachangelog.com/en/1.0.0/)

## 0.1.0 (Unreleased)

### Added
 - Version bumps to Go, Terraform and UpCloud?
 - Started a Changelog for future reference

### Changed
 - smthg smthg

### Removed
 - smthg smthg

...
```

Reply:
Yes good call and moving forward we should make it a required change in other PR for the provider.  I will flesh this out 👍

Reply:
I have updated with an initial set of alterations.